src/style.css

```diff
@@ -0,0 +1,28 @@
+body {
+  background-color: #000;
+  margin: 0;
+}
+
+.block {
+  max-width: 350px;
```

Comment:
you add this class to every block, but only `blue` one should have width of 350px; Others are smaller